data/data/agent/files/usr/local/bin/set-node-zero.sh

```diff
@@ -45,6 +45,16 @@ done
 if [ "${IS_NODE_ZERO}" = "true" ]; then
     echo "Node 0 IP ${NODE_ZERO_IP} found on this host" 1>&2
 
+    # Create tls certs, if they don't exist, via the installer command.
+    # This allows the certs to be created at run-time, e.g. when installed via the UI
+    AGENT_TLS_DIR=/opt/agent/tls
+    if [ -z $(ls -A "$AGENT_TLS_DIR") ]; then
+       . /usr/local/bin/release-image.sh
```

Comment:
It could be worth echoing a message, to leave a trace in the journal that the certs have been lazily generated (for troubleshooting)

Reply:
Presumably the installer will log something.